docs/download.md

```diff
@@ -18,6 +18,63 @@ cuDF jar, that is either preinstalled in the Spark classpath on all nodes or sub
 that uses the RAPIDS Accelerator For Apache Spark. See the [getting-started
 guide](https://nvidia.github.io/spark-rapids/Getting-Started/) for more details.
 
+## Release v22.04.0
+Hardware Requirements:
+
+The plugin is tested on the following architectures:
+
+	GPU Models: NVIDIA V100, T4 and A2/A10/A30/A100 GPUs
+
+Software Requirements:
+
+	OS: Ubuntu 18.04, Ubuntu 20.04 or CentOS 7, CentOS 8
+
+	CUDA & NVIDIA Drivers*: 11.x & v450.80.02+
+
+	Apache Spark 3.1.1, 3.1.2, 3.1.3, 3.2.0, 3.2.1, 3.3.0, Cloudera CDP 7.1.6, 7.1.7, Databricks 9.1 ML LTS or 10.4 ML LTS Runtime and GCP Dataproc 2.0
```

Comment:
what are we doing with CDP here, we don't have the code in our shim to actually work with those.
@sameerz   Should just point to their release info ?

Reply:
Should we completely remove CDP versions from download page like what we did for EMR before?

How about we add the CDP version in FAQ with below 2 links?
https://docs.cloudera.com/cdp-private-cloud-base/7.1.6/yarn-allocate-resources/topics/yarn-using-gpu-scheduling.html
https://docs.cloudera.com/cdp-private-cloud-base/7.1.7/yarn-allocate-resources/topics/yarn-using-gpu-scheduling.html

Reply:
After some discussion, i added a section in FAQ guide for CDP/CDS and also removed from download page. Please take it look at again to see if it looks good to you?